platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/PlatformState.java

```diff
@@ -83,56 +230,290 @@ public PlatformState copy() {
+        this.addressBook = Objects.requireNonNull(addressBook);
+    }
+
+    /**
+     * Get the previous address book.
      */
-    public void setAddressBook(final AddressBook addressBook) {
-        setChild(ChildIndices.ADDRESS_BOOK, addressBook);
+    @Nullable
+    public AddressBook getPreviousAddressBook() {
+        return previousAddressBook;
     }
 
     /**
-     * Get the object containing miscellaneous round information.
+     * Set the previous address book.
      *
-     * @return round data
+     * @param addressBook an address book
      */
-    public PlatformData getPlatformData() {
-        return getChild(ChildIndices.PLATFORM_DATA);
+    public void setPreviousAddressBook(@Nullable final AddressBook addressBook) {
+        this.previousAddressBook = addressBook;
     }
 
     /**
-     * Set the object containing miscellaneous platform information.
+     * Get the round when this state was generated.
      *
-     * @param round
-     * 		round data
+     * @return a round number
      */
-    public void setPlatformData(final PlatformData round) {
-        setChild(ChildIndices.PLATFORM_DATA, round);
+    public long getRound() {
+        return round;
     }
 
     /**
-     * Get the previous address book.
+     * Set the round when this state was generated.
+     *
+     * @param round a round number
      */
-    public AddressBook getPreviousAddressBook() {
-        return getChild(ChildIndices.PREVIOUS_ADDRESS_BOOK);
+    public void setRound(final long round) {
+        this.round = round;
     }
 
     /**
-     * Set the previous address book.
+     * Get the running hash of all events that have been applied to this state since the beginning of time.
+     *
+     * @return a running hash of events
+     */
+    public Hash getHashEventsCons() {
+        return hashEventsCons;
+    }
+
+    /**
+     * Set the running hash of all events that have been applied to this state since the beginning of time.
+     *
+     * @param hashEventsCons a running hash of events
+     */
+    public void setHashEventsCons(final Hash hashEventsCons) {
+        this.hashEventsCons = hashEventsCons;
+    }
+
+    /**
+     * Get the consensus timestamp for this state, defined as the timestamp of the first transaction that was applied in
+     * the round that created the state.
+     *
+     * @return a consensus timestamp
+     */
+    public Instant getConsensusTimestamp() {
+        return consensusTimestamp;
+    }
+
+    /**
+     * Set the consensus timestamp for this state, defined as the timestamp of the first transaction that was applied in
+     * the round that created the state.
+     *
+     * @param consensusTimestamp a consensus timestamp
+     */
+    public void setConsensusTimestamp(final Instant consensusTimestamp) {
+        this.consensusTimestamp = consensusTimestamp;
+    }
+
+    // TODO remove methods that are not simple getters and setters
```

Comment:
TODO left in code.

Reply:
These TODOs should be done in future PRs, not this one. Removed them for now.